arch/lkl/auto.conf

```diff
@@ -1 +1,8 @@
-export OUTPUT_FORMAT=$(shell $(LD) -r -print-output-format)
+ifneq ($(shell $(LD) --version 2>&1 | head -n 1 | grep LLD),)
+  CC_TARGET=$(word 2,$(shell $(CC) --version | grep Target))
+  ifeq ($(CC_TARGET),x86_64-unknown-linux-gnu)
+    export OUTPUT_FORMAT=elf64-x86-64
```

Comment:
Can you please rebase to latest master? I just merged a change that will require reworking this patch.

Reply:
I rebased and fixed the patch 2b07681.